src/callstack.cr

```diff
@@ -23,6 +23,10 @@ end
 
 # :nodoc:
 struct CallStack
+  # Compute current directory at the beginning so filenames
+  # are always shown relative to the *starting* working directory.
+  CURRENT_DIR = Process::INITIAL_PWD + '/'
```

Comment:
Same here: `File::SEPARATOR`.

Reply:
We really should have a function to get a path relative to another path
https://docs.python.org/3/library/os.path.html#os.path.relpath

Reply:
@oprypin I agree, though I don't know if I'd like `../../` to appear in the output. I checked Go for example and they always use absolute paths (though Ruby uses relative paths if the path is in the current directory)